lang/llvm-10/Portfile

```diff
@@ -0,0 +1,618 @@
+name                    llvm-${llvm_version}
+revision                0
+subport                 clang-${llvm_version} { revision 0 }
+subport                 lldb-${llvm_version} { revision 0 }
+set suffix              mp-${llvm_version}
+set sub_prefix          ${prefix}/libexec/llvm-${llvm_version}
+dist_subdir             llvm
+categories              lang
+platforms               darwin
+license                 NCSA
+maintainers             {jeremyhu @jeremyhu} {larryv @larryv} {kencu @kencu}
+
+if {${subport} eq "llvm-${llvm_version}"} {
+    homepage            https://llvm.org/
+    description         llvm is a next generation compiler infrastructure
+    long_description    The LLVM Core libraries provide a modern source- and \
+                        target-independent optimizer, along with code \
+                        generation support for many popular CPUs (as well as \
+                        some less common ones!) These libraries are built \
+                        around a well specified code representation known as \
+                        the LLVM intermediate representation ("LLVM IR").
+
+    depends_lib         port:libedit port:libffi port:ncurses path:lib/libxar.dylib:xar port:zlib
+    depends_run         bin:perl:perl5 port:llvm_select
+} elseif {${subport} eq "clang-${llvm_version}"} {
+    homepage            https://clang.llvm.org/
+    description         C, C++, Objective C and Objective C++ compiler
+    long_description    Clang is an "LLVM native" C/C++/Objective-C compiler, \
+                        which aims to deliver amazingly fast compiles (e.g. \
+                        about 3x faster than GCC when compiling Objective-C \
+                        code in a debug configuration), extremely useful error \
+                        and warning messages and to provide a platform for \
+                        building great source level tools. The included Clang \
+                        Static Analyzer is a tool that automatically finds bugs in \
+                        your code, and is a great example of the sort of tool \
+                        that can be built using the Clang frontend as a \
+                        library to parse C/C++ code.
+
+    depends_lib         port:libxml2 port:libomp port:llvm-${llvm_version}
+    depends_run         port:clang_select port:ld64
+    depends_skip_archcheck-append ld64 subversion
+
+    # Avoid requiring a bootstrap version of perl5 on 10.6.
+    if {${os.major} >= 11} {
+        default_variants    +analyzer
+    }
+
+} elseif {${subport} eq "lldb-${llvm_version}"} {
+    homepage            https://lldb.llvm.org/
+    description         the LLVM debugger
+    long_description    Lldb is the "LLVM native" debugger.
+
+    depends_lib         port:libxml2 port:libomp port:llvm-${llvm_version} port:ncurses
+    depends_build-append port:swig-python port:doxygen
+}
+
+version                 ${llvm_version}.0.0
+epoch                   1
```

Comment:
As this is a brand new set of ports, the epoch does not need to be set, or ca be set to 0

Reply:
You're right -- that's just been carried over from previous llvm's for I don't know how long...